akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md

```diff
@@ -50,8 +50,33 @@ Use plain `system.actorOf` instead of the DSL to create Actors if you have been
 ### TypedActor
 
 `akka.actor.TypedActor` has been deprecated as of 2.6 in favor of the
-`akka.actor.typed` API which should be used instead. For more context on
-this deprecation [please refer to the discussion here](https://github.com/akka/akka/issues/26144).
+`akka.actor.typed` API which should be used instead.
+
+The reason for phasing out the old `TypedActor` is mostly that it doesn't match our recommended way
+of implementing and interacting with actors. They use transparent remoting, which we believe is a bad idea.
+Transparent remoting is when you try to make remote method invocations look like local calls. In contrast we
+believe in location transparency with explicit messaging between actors (same type of messaging for both local
+and remote actors).
```

Comment:
```suggestion
There are several reasons for phasing out the old `TypedActor`. The primary reason is it doesn't match our recommended way of implementing and interacting with actors because they use transparent remoting, which we believe is a bad idea. Transparent remoting is when you try to make remote method invocations look like local calls. In contrast we believe in location transparency with explicit messaging between actors (same type of messaging for both local and remote actors). Additionally, they have limited functionality compared to ordinary actors and worse performance.
```